clients/include/testing_rotg_strided_batched.hpp

```diff
@@ -147,7 +148,6 @@ void testing_rotg_strided_batched(const Arguments& arg)
 
             if(arg.unit_check)
             {
-                double rel_error = std::numeric_limits<U>::epsilon() * 100;
```

Comment:
Looks like the scalar here was 100 before, this error wasn't enough for hipcc or is it just changed for consistency?